Bio/Graphics/GenomeDiagram/_Feature.py

```diff
@@ -129,7 +129,7 @@ def __process_feature(self):
         """Examine wrapped feature and set some properties accordingly (PRIVATE)."""
         self.locations = []
         bounds = []
-        # This will be a list of length one for simple FeatureLocation:
+        # This will be a list of length one for simple SimpleLocation:
```

Comment:
drop simple here

Reply:
Yes, can do. That was done on your branch, doing it here too ought not to complicate the merge/rebase.